src/_P103_Atlas_EZO_pH_ORP_EC.ino

```diff
@@ -0,0 +1,641 @@
+                                    F("Plugin_103_ref_cal_H"),
+                                    PCONFIG_FLOAT(3),
+                                    0,
+                                    14,
+                                    2,
+                                    0.01);
+
+            if (nb_calibration_points > 2) {
+                addHtml(F("&nbsp;<span style='color:green;'>OK</span>"));
+            } else {
+                addHtml(F("&nbsp;<span style='color:orange;'>Not yet calibrated</span>"));
+            }
+            addFormCheckBox(F("Enable"), F("Plugin_103_enable_cal_H"), false);
+            addHtml(F(
+                        "\n<script type='text/javascript'>document.getElementById(\"Plugin_103_enable_cal_H\").onclick = function(){document.getElementById(\"Plugin_103_enable_cal_L\").checked = false;document.getElementById(\"Plugin_103_enable_cal_M\").checked = false;};</script>\n"));
+
+            if (nb_calibration_points > 1) {
+                char sensordata[32];
+                bool status;
+                status = _P103_send_I2C_command(I2Cchoice, F("Slope,?"), sensordata);
+
+                if (status) {
+                String slopeAnswer = F("Answer to 'Slope' command : ");
+                slopeAnswer += sensordata;
+                addFormNote(slopeAnswer);
+                }
+            }
+            break;
+          }
+        
+        case ORP:
+        {
+            int nb_calibration_points = -1;
+            status = _P103_send_I2C_command(I2Cchoice, F("Cal,?"), sensordata);
+
+            if (status) {
+                if (strncmp(sensordata, "?Cal,", 5)) {
+                char tmp[2];
+                tmp[0]                = sensordata[5];
+                tmp[1]                = '\0',
+                nb_calibration_points = atoi(tmp);
+                }
+            }
+
+            addRowLabel(F("<strong>ORP Calibration</strong>"));
+            addFormFloatNumberBox(F("Ref ORP"), F("Plugin_103_ref_cal_O"), PCONFIG_FLOAT(1), 0, 1500, 1, 1);
+
+            if (nb_calibration_points > 0) {
+                addHtml(F("&nbsp;<span style='color:green;'>OK</span>"));
+            } else {
+                addHtml(F("&nbsp;<span style='color:red;'>Not yet calibrated</span>"));
+            }
+            addFormCheckBox(F("Enable"), F("Plugin_103_enable_cal_O"), false);
+            break;
+        }
+
+        case EC:
+        {
+            addRowLabel(F("<strong>Dry calibration</strong>"));
+            addFormCheckBox(F("Enable"), F("Plugin_103_enable_cal_dry"), false);
+            addHtml(F(
+                        "\n<script type='text/javascript'>document.getElementById(\"Plugin_103_enable_cal_dry\").onclick=function() {document.getElementById(\"Plugin_103_enable_cal_single\").checked = false;document.getElementById(\"Plugin_103_enable_cal_L\").checked = false;;document.getElementById(\"Plugin_103_enable_cal_H\").checked = false;};</script>"));
```

Comment:
See `html_add_script` and `html_add_script_end`

Reply:
ok thx.